src/Sulu/Bundle/ContentBundle/Search/EventListener/ReindexListener.php

```diff
@@ -94,59 +110,99 @@ public function onIndexRebuild(IndexRebuildEvent $event)
             'SELECT * FROM [nt:unstructured] AS a WHERE ' . $condition
         );
 
-        $count = [];
 
+        $batchSize = 50;
+        $offset = $this->resumeManager->getCheckpoint(self::CHECKPOINT_NAME, 0);
+        $batchLimit = null;
+        $batchCount = 0;
+
+        do {
```

Comment:
If it is not too much overhead, we should show the number of documents to process..